### DISCUSSION TITLE
Title page with two side span possible?

### DISCUSSION THREAD

**Discussion by hannahmonta** [nmfs-opensci/quarto_titlepages#40] [Q&A]
Hi,
Thanks for this great explanation on how to set up individual title pages for quarto.
I just used it and it is great! I have one issue that I could not solve with this extension, maybe you could help me out?

I want the titlepage to span for two pages (I have to add information on my supervisors etc and it should appear on a secon title page). Is it possible to include a pagebreak? I tried it but I did not get it worked out. The text I want to appear on page 2 of the title page just disappeared at all. Really appreciate your help!

Best, Hannah

**Comment by eeholmes** [comment]
Hi, I think the way to do this with the least headaches is a static tex file after the titlepage:
https://nmfs-opensci.github.io/quarto_titlepages/06-static-files.html#copyright-dedication-and-other-static-pages

Something along these lines in `page2.tex`. Note I didn't test but it should be something along these lines. 

```
\newpage
\thispagestyle{plain}
The tex for page 2 of my title page
\clearpage
```

**Comment by eeholmes** [comment]
Note moving this to discussion Q&A so others can search it in the future.
